figshare/Utils.py

```diff
@@ -248,20 +248,28 @@ def calculate_ual_rdm_size(config, article_id: int, version: str):
     article_version_ual_rdm = ""
     version_ual_rdm_size = 0
     curation_storage = config['curation_storage_location']
-    if os.access(curation_storage, os.R_OK):
+    if os.path.exists(curation_storage) and os.access(curation_storage, os.R_OK):
         curation_storage_items = os.scandir(curation_storage)
         for item in curation_storage_items:
             if item.is_dir() and item.name.__contains__(str(article_id)):
                 article_dir = os.path.join(curation_storage, item.name)
                 break
+        if not os.path.exists(article_dir):
+            return 0
```

Comment:
In this case, "0" does not mean zero bytes, it means that the directory was not found, correct? If so, please add a note in the function documentation `return` statement explaining the new behavior and the different meanings "0" could have

Reply:
Yes, zero bytes since the function is expected to return an int. The 0 does not represent Boolean False.